.github/workflows/build-modules.yml

```diff
@@ -33,24 +33,10 @@ jobs:
         env:
           RUSTFLAGS: ""
         run: |
-          features=("buffer"
-                    "child-process"
-                    "crypto"
-                    "events"
-                    "exceptions"
-                    "fs"
-                    "navigator"
-                    "net"
-                    "os"
-                    "path"
-                    "process"
-                    "perf-hooks"
-                    "timers"
-                    "url"
-                    "zlib")
-          for feature in "${features[@]}"; do
-            echo "Compiling feature: $feature"
-            cargo build -p llrt_modules --no-default-features --features $feature
+          readarray crates < <(cargo metadata --no-deps --format-version 1 --quiet | jq -r '.packages[] | select(.manifest_path | contains("modules/")) | .name')
+          for crate in "${crates[@]}"; do
+            echo "Compiling feature: $crate"
+            cargo build -p $crate
```

Comment:
Isn't this used in `compile.sh`script?

Reply:
I will check

Reply:
Thats a file I created for local testing, dont need it